tf-modules/cumulus/variables.tf

```diff
@@ -1,5 +1,11 @@
 # Required
 
+variable "async_operation_image" {
+  description = "docker image to use for Cumulus async operations tasks"
+  type = string
+  default = "cumuluss/async-operation:27"
```

Comment:
Should this get updated to 32 now also?  I see they're over written in the example variables.tf var.  Wasn't sure though.

Reply:
I assume it's getting overwritten by the declaration in example/cumulus-tf/variables.tf, but should they match?

Reply:
Yeah, they should, I'll update that.

Reply:
Actually, this already *was* updated: https://github.com/nasa/cumulus/pull/2024/commits/7d1972c24bcd9e6580183c11a0a5992052c8b093

Reply:
Fixed, thanks for the feedback.   6fac2303e